apigee_devportal_kickstart.info.yml

```diff
@@ -57,9 +61,12 @@ install:
 - components
 - fontawesome
 - forum
+- paragraphs
+- pathauto
 
 # List any themes that should be installed as part of the profile installation.
 # Note that this will not set any theme as the default theme.
 themes:
+- apigee_kickstart
```

Comment:
THIS IS IT!  Mind you, I had this originally, and at some point it failed (configuration import), though it must have failed for other reasons (problems with my config somewhere).  This makes 💯 sense now, given what I've learned, which I want to add here, for my future self.

- https://www.drupal.org/node/2453919 via @arshad: 
    > All configuration contained within a module's, theme's or profile's config/install must be created during an install. If any configuration cannot be created because of missing dependencies then a `Drupal\Core\Config\UnmetDependenciesException` exception is thrown. At this point the module or profile author can either, add the missing dependency to the list of dependencies in the extension's `.info.yml` file, or, move the configuration to `config/optional`. Themes can not directly depend on modules so all configuration entities created by themes should go in `config/optional`.
- Optional Config Weirdness in Drupal 8
https://dev.acquia.com/blog/optional-config-weirdness-in-drupal-8/09/11/2017/18866
- From hook_install_tasks() https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_install_tasks
    > Any tasks you define here will be run, in order, *after the installer has finished the site configuration step* but *before it has moved on to the final import of languages and the end of the installation*. This is invoked by install_tasks().